gitman/models/source.py

```diff
@@ -220,7 +220,7 @@ def identify(self, allow_dirty=True, allow_missing=True, skip_changes=False):
 
         raise self._invalid_repository
 
-    def lock(self, rev=None, allow_dirty=False, skip_changes=False):
+    def lock(self, rev=None, allow_dirty=False, skip_changes=False, verify_rev=True):
```

Comment:
Is there any easy way to make this default to `False` to avoid potentially breaking changes?

Reply:
Could add a top-level flag `--verify` and default to False if that's acceptable.

Reply:
Though it will only verify *if* the user specifies the source in the format `sourcename@rev`, so I don't anticipate any breaking changes.

Reply:
I don't think we need to go that far yet, but instead only pass `verify_rev=True` if the `rev` value from the CLI originally contained `@`, which was later parsed.

Reply:
I think the point I was trying to make, was that when `rev==None`, then `verify_rev` will be ignored. Since the calling function `lock_dependencies` sets `rev=None` if `@` sign isn't present then as long as the user isn't using `sourcename@rev` it doesn't matter what `verify_rev` is set to.

I think maybe this behavior is confusing, I'm open to suggestions on how to improve it.

Reply:
In that case, I think this implementation works!